examples/visual-tests/amp-story/amp-story-page-attachment.html

```diff
@@ -192,6 +192,9 @@ <h1>Some really cool video</h1>
       </amp-story-page>
 
       <amp-story-page id="attachment-with-form">
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+        </p>
```

Comment:
I think it's okay to leave this out since it's unrelated to the component.

Reply:
Removed! I originally put this in so that I could programmatically dismiss the attachment by tapping it, but I think a better alternative is getting the test to just tap the spacer element instead. I'll fix the test if it starts failing

Reply:
Looks like the spacer element isn't visible on the desktop version of the test. I've added a small paragraph element with the text "Element used for attachment dismissal" that can be tapped for dismissal on both desktop on mobile. Thoughts?

Reply:
We could trigger a click on `.i-amphtml-story-page-attachment-close-button`. 
It's built on both mobile and desktop.

Reply:
Great, thanks! Updated :)